locations/spiders/standard_bank_za.py

```diff
@@ -0,0 +1,160 @@
+import scrapy
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_3_LETTERS, OpeningHours
+from locations.pipelines.address_clean_up import clean_address
+
+ZA_PROVINCES = [
+    "Eastern Cape",
+    "Free State",
+    "Gauteng",
+    "KwaZulu-Natal",
+    "Limpopo",
+    "Mpumalanga",
+    "North West",
+    "Northern Cape",
+    "Western Cape",
+]
```

Comment:
Should this go in a central file somewhere? I wasn't quite sure of the best file, or if I should make a new file

Reply:
There isn't one specifically for these yet, as most spiders that search by these grab them from a dropdown or similar.

The closest spot would be locations/searchable_points/ which has subfolders for postcodes, and a bunch of lat/long datt.

Maybe wait until you have a few spiders with the same or similar list; then extract?